### PR TITLE
[DOC] Update repo path

### PR DESCRIPTION
**Pull request by skoudoro**
- Update repo path a bit everywhere before the announcement.
- Add azure badge

### CODE REVIEW

**Comment by skoudoro** [comment]
it seems that Travis has been triggered correctly as you can see [here](https://travis-ci.org/dipy/dipy) but it does not show up here.... strange...

**Comment by skoudoro** [comment]
yeah, Travis is here!  All green, This PR is ready to go! Does anyone want to have a look 😄?

File: doc/release_notes/release0.12.rst [hunk 1]
```diff
@@ -462,7 +462,7 @@ Issues (342):
 * :ghissue:`1035`: Fix for fetcher files in Windows
 * :ghissue:`1016`: viz.fvtk has no attribute 'ren'
 * :ghissue:`1033`: Viz surfaces
-* :ghissue:`1032`: Merge pull request #1 from nipy/master
```
Comment (arokem) [inline doc/release_notes/release0.12.rst]: I don't think you want to change this!

File: doc/release_notes/release0.6.rst [hunk 2]
```diff
@@ -121,7 +121,7 @@ Pull Requests (100):
 * :ghpull:`24`: TST: Fixing tests in reconst module.
 * :ghpull:`27`: DOC - Add reference to white matter diffusion values.
 * :ghpull:`25`: NF - Add prolate white matter as defaults for multi-tensor signal sim.
-* :ghpull:`22`: Updating my fork with the nipy master
```
Comment (arokem) [inline doc/release_notes/release0.6.rst]: I don't think you want to change this

File: doc/release_notes/release0.6.rst [hunk 3]
```diff
@@ -245,7 +245,7 @@ Issues (125):
 * :ghissue:`24`: TST: Fixing tests in reconst module.
 * :ghissue:`27`: DOC - Add reference to white matter diffusion values.
 * :ghissue:`25`: NF - Add prolate white matter as defaults for multi-tensor signal sim.
-* :ghissue:`22`: Updating my fork with the nipy master
```
Comment (arokem) [inline doc/release_notes/release0.6.rst]: I don't think you want to change this

**Comment by skoudoro** [comment]
Good catch, thanks! I just updated it!

**Comment by arokem** [comment]
OK  - I think that this is ready to go.

**Comment by skoudoro** [comment]
Thank you!

**Pull request merged**